test/fixtures/user_fixtures.js

```diff
@@ -2,12 +2,161 @@
+  },
+  {
+    name: 'tokens:update',
+    description: 'Generatekey'
+  },
+  {
+    name: 'tokens:delete',
+    description: 'Revokekey'
+  },
+  {
+    name: 'transactions:read',
+    description: 'Viewtransactionslist'
+  },
+  {
+    name: 'transactions-by-date:read',
+    description: 'Searchtransactionsbydate'
+  },
+  {
+    name: 'transactions-by-fields:read',
+    description: 'Searchtransactionsbypaymentfields'
+  },
+  {
+    name: 'transactions-download:read',
+    description: 'Downloadtransactions'
+  },
+  {
+    name: 'transactions-details:read',
+    description: 'Viewtransactiondetails'
+  },
+  {
+    name: 'transactions-events:read',
+    description: 'Viewtransactionevents'
+  },
+  {
+    name: 'refunds:create',
+    description: 'Issuerefund'
+  },
+  {
+    name: 'transactions-amount:read',
+    description: 'Viewtransactionamounts'
+  },
+  {
+    name: 'transactions-description:read',
+    description: 'Viewtransactiondescription'
+  },
+  {
+    name: 'transactions-email:read',
+    description: 'Viewtransactionemail'
+  },
+  {
+    name: 'transactions-card-type:read',
+    description: 'Viewtransactioncardtype'
+  },
+  {
+    name: 'gateway-credentials:read',
+    description: 'Viewgatewayaccountcredentials'
+  },
+  {
+    name: 'gateway-credentials:update',
+    description: 'Editgatewayaccountcredentials'
+  },
+  {
+    name: 'service-name:read',
+    description: 'Viewservicename'
+  },
+  {
+    name: 'service-name:update',
+    description: 'Editservicename'
+  },
+  {
+    name: 'payment-types:read',
+    description: 'Viewpaymenttypes'
+  },
+  {
+    name: 'payment-types:update',
+    description: 'Editpaymenttypes'
+  },
+  {
+    name: 'email-notification-template:read',
+    description: 'Viewemailnotificationstemplate'
+  },
+  {
+    name: 'email-notification-paragraph:update',
+    description: 'Editemailnotificationsparagraph'
+  },
+  {
+    name: 'email-notification-toggle:update',
+    description: 'Turnemailnotificationson/off'
+  },
+  {
+    name: 'tokens:read',
+    description: 'View keys'
+  },
+  {
+    name: 'toggle-3ds:read',
+    description: 'View 3D Secure setting'
+  },
+  {
+    name: 'toggle-3ds:update',
+    description: 'Edit 3D Secure setting'
+  },
+  {
+    name: 'users-service:delete',
+    description: 'Remove user from a service'
+  },
+  {
+    name: 'merchant-details:read',
+    description: 'View Merchant Details setting'
+  },
+  {
+    name: 'merchant-details:update',
+    description: 'Edit Merchant Details setting'
+  },
+  {
+    name: 'toggle-billing-address:read',
+    description: 'View Billing Address setting'
+  },
+  {
+    name: 'toggle-billing-address:update',
+    description: 'Edit Billing Address setting'
+  },
+  {
+    name: 'go-live-stage:update',
+    description: 'Update Go Live stage'
+  },
+  {
+    name: 'go-live-stage:read',
+    description: 'View Go Live stage'
+  }
+]
+
 // Setup
 const pactUsers = pactBase(
   {
```

Comment:
Tiny thing but this if this was 

```js 
const pactUsers = pactBase({ 
  array: ...
  length: ... 
})
```

it would be easier to read

Reply:
Done